en/advanced_config/prearm_arm_disarm.md

```diff
@@ -4,13 +4,60 @@ Vehicles may have moving parts, some of which are potentially dangerous when pow
+> **Note** Ground stations may display *disarmed* for both disarmed and pre-armed states.
+  While not technically correct for pre-armed vehicles, it is "safe".
+  
+A [safety switch](../getting_started/px4_basic_concepts.md#safety_switch) (if available) is used as a precondition for arming the vehicle, and may also be used to enable the pre-armed state.
 
-PX4 allows you to configure how prearming, arming and disarming work using parameters (which can be edited in *QGroundControl* via the [parameter editor](../advanced_config/parameters.md)), as described in the following sections.
+When permitted, arming is enabled using an [arming gesture](#arm_disarm_gestures), [arming switch](#arm_disarm_switch) or MAVLink command. 
+
+PX4 allows you to configure how pre-arming, arming and disarming work using parameters (which can be edited in *QGroundControl* via the [parameter editor](../advanced_config/parameters.md)), as described in the following sections.
+
+> **Note** Arming/disarming parameters can be found in [Parameter Reference > Commander](../advanced_config/parameter_reference.md#commander) (search for `COM_ARM_` and `COM_DISARM_*`).
+
+## Arming Gesture {#arm_disarm_gestures}
+
+By default, the vehicle is armed and disarmed by setting RC throttle/yaw to particular values and holding for 1 second
+- **Arming:** Throttle minimum, yaw maximum
+- **Disarming:** Throttle minimum, yaw minimum
+
+RC controllers will have different gestures [based on their mode](../getting_started/rc_transmitter_receiver.md#types-of-remote-controls) (as controller mode affects the sticks used for throttle and yaw):
+- **Mode 2**: 
+  - *Arm:* Left stick to bottom right. 
+  - *Disarm:* Left stick to the bottom left.
+- **Mode 1**: 
+  - *Arm:* Left-stick to right, right-stick to bottom.
+  - *Disarm:* - Left-stick to left, right-stick to the bottom.
+
+The required hold time can be configured using [COM_RC_ARM_HYST](#COM_RC_ARM_HYST).
+
+Parameter | Description
+--- | ---
+<span id="COM_RC_ARM_HYST"></span>[COM_RC_ARM_HYST](../advanced_config/parameter_reference.md#COM_RC_ARM_HYST) | Time that RC stick must be held in arm/disarm position before arming/disarming occurs (default: 1 second).
+
+## Arming Button/Switch {#arm_disarm_switch}
+
+An *arming button* or "momentary switch" can be configured to trigger arm/disarm *instead* of [gesture-based arming](#arm_disarm_gestures) (setting an arming switch disables arming gestures).
+The button should be held down for ([nominally](#COM_RC_ARM_HYST)) one second to arm (when disarmed) or disarm (when armed).
+
+A two-position switch can also be used for arming/disarming, where the respective arm/disarm commands are sent on switch *transitions*.
+
+> **Tip** Two-position arming switches are primarily used in/recommended for racing drones.
+
+is not recommended because it is easy to get the switch into an inconsistent state (i.e. if arming is rejected, the switch will stay armed even though the vehicle is not armed).
```

Comment:
```suggestion
The use of a two-position switch if not explicitly required is not recommended because it is easy to get the switch into an inconsistent state (i.e. if arming is rejected, the switch will stay armed even though the vehicle is not armed).
```

Reply:
Whoops. That bit is supposed to be deleted.